docs/config.md

```diff
@@ -1343,12 +1343,12 @@ The above configuration snippet sets 2 cpus for the processes annotated with the
 
 #### Selector priority
 
-When mixing generic process configuration and selectors the following priority rules are applied (from lower to higher):
+Process configuration settings are resolved as follows (from lowest to highest priority):
```

Comment:
can you flip the list from highest to lowest priority, for consistency with the other lists? (and hence improved overall readability)

Reply:
I think I actually prefer going from lowest to highest, I find it easier to reason about. What if we switched the other lists to be low to high?

Reply:
Switching the other lists, let me know what you think

Reply:
Agree